contributing/getting-started-quick.md

```diff
@@ -48,7 +48,7 @@ This Quick Start guide is the TL;DR version of the longer [end-to-end guide](get
 * Go to the branch: `git checkout <branch name>`
 
 # Build AMP & run a local server
-* Make sure you have the latest packages (after you pull): `yarn`
+* Make sure you have the latest packages (after you pull): `npm install`
```

Comment:
Another useful command is to run `npm prune` to make sure unnecessary dependencies are removed.

Reply:
Added it below in the "Create commits to contain your changes" section